### ISSUE TITLE
"AIRMap" was not found in the UIManager IN "0.26.0"

### ISSUE THREAD

**Issue by siayara** [react-native-community/react-native-maps#3099]
### Is this a bug report?

(write your answer here)

### Have you read the [Installation Instructions](https://github.com/react-community/react-native-maps/blob/master/docs/installation.md)?

(Write your answer here.)

### Steps to Reproduce

(Write your steps here:)

1. 2. 3.

### Expected Behavior

(Write what you thought would happen.)

### Actual Behavior

(Write what happened. Add screenshots!)

### Reproducible Demo

(Paste the link to an example project and exact instructions to reproduce the issue.)

**Comment by betoharres** [comment]
Yeah the issue on is not filled but I also got this error by **just** updating the library, everything worked fine on 0.25.0 though - tested on **Android**, not tested on iOS yet.

A little disclaimer is that I updated my react-native project manually from 0.59 to 0.60.5 and to 0.61.1 a few days ago, so I'm not sure if that could be related.
here's my react-native info output:
```
System:
    OS: Linux 5.1 Arch Linux undefined
    CPU: (6) x64 Intel(R) Core(TM) i5-8400 CPU @ 2.80GHz
    Memory: 2.63 GB / 15.59 GB
    Shell: 5.7.1 - /usr/bin/zsh
  Binaries:
    Node: 11.15.0 - /usr/bin/node
    Yarn: 1.16.0 - /usr/bin/yarn
    npm: 6.10.0 - /usr/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    Android SDK:
      API Levels: 26, 28, 29
      Build Tools: 26.0.3, 28.0.3, 29.0.0
      System Images: android-28 | Google Play Intel x86 Atom, android-29 | Google APIs Intel x86 Atom
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.1 => 0.61.1 
  npmGlobalPackages:
    react-native-cli: 2.0.1
```

**Comment by kneza23** [comment]
Yeah, the same thing happens with me

**Comment by kefyru** [comment]
In IOS works, but in Android i've got this error: "AIRMap" was not found in the UIManager

**Comment by AriaEdo** [comment]
Downgrade to 0.25.0 works fine

**Comment by kefyru** [comment]
> Downgrade to 0.25.0 works fine
on which rn version you checked?

**Comment by mahajan8** [comment]
Any other solution than downgrading?
Stuck with this issue on android. IOS works fine.

**Comment by mahajan8** [comment]
#3081 
Maybe this can help, maybe not, i just found this. You could try too.
I'll update if something works with me.

**Comment by wtfkeklol** [comment]
> Downgrade to 0.25.0 works fine

how do you properly downgrade react-native-maps?

i did 
`npm uninstall react-native-maps`
then
` npm install react-native-maps@0.25.0`

but now I get an error that says 
`"Unable to resolve module `./lib/components/MapHeatmap.js` from `node_modules\react-native-maps\index.js`" `
when it's not even referenced in the index.js file. 
I guess something got stuck, but I have no idea, never had to downgrade any packages.

**Comment by dl-husky73** [comment]
I just upgraded and received the same error.   Any fixes yet?

**Comment by skippone** [comment]
I believe the problem lies in `react-native.config.js` file for this package.

Currently it contains this configuration:

```
module.exports = {
  project: {
    ios: {},
    android: {
      sourceDir: './lib/android',
    },
   },
 };
```

while the valid configuration should looks like this:

```
module.exports = {
  dependency: {
      platforms: {
          android: {
              sourceDir: './lib/android',
          },
      },
  },
};

```

Apparently when RN configuration was moved from project.json to separate file it was not converted to new specification.

**Comment by dl-husky73** [comment]
Thanks Skippone, that worked.   Just made the change, reset and relaunched and maps working great!  Appreciate the quick and more importantly, accurate reply!

**Comment by rborn** [comment]
@skippone mind a PR? 😻

**Comment by rizwanatta** [comment]
i have had that issue solved doing rborn what just did! tested on both devices it works like a charm if the config file is edited as done in the PR I sent!

**Comment by skippone** [comment]
> @skippone mind a PR? 😻

Well, someone was already faster than me :)

**Comment by rborn** [comment]
@rizwanatta @skippone 😻

@christopherdro can we make an "emergency release" with this fix? I just merged 🤗

**Comment by whiskyplausible** [comment]
> 
> 
> > Downgrade to 0.25.0 works fine
> 
> how do you properly downgrade react-native-maps?
> 
> i did
> `npm uninstall react-native-maps`
> then
> ` npm install react-native-maps@0.25.0`
> 
> but now I get an error that says
> `"Unable to resolve module `./lib/components/MapHeatmap.js`from`node_modules\react-native-maps\index.js`" `
> when it's not even referenced in the index.js file.
> I guess something got stuck, but I have no idea, never had to downgrade any packages.

I'm also getting this error and can't see any way to fix it. I think the problem might be happening with the react-google-maps module

**Comment by NoumanSakhawat** [comment]
anyone tell me how to handle this problem i have the same issue

**Comment by rizwanatta** [comment]
the issue is closed! as it is been solved in the PR! and will be soon released with the new version! @NoumanSakhawat @whiskyplausible

**Comment by rizwanatta** [comment]
@whiskyplausible downgrading is not an active solution! I have had been in the mindset of that too! I will say: you see the answer here of @skippone ! this should help you! use the present release for now!

**Comment by NoumanSakhawat** [comment]
> the issue is closed! as it is been solved in the PR! and will be soon released with the new version! @NoumanSakhawat @whiskyplausible

How much time it will take please let me know?

**Comment by rborn** [comment]
@rizwanatta please use the master for now, will be published as soon as @christopherdro has some time to look into :)

**Comment by rborn** [comment]
closing because https://github.com/react-native-community/react-native-maps/releases/tag/v0.26.1  🎉

**Comment by eduardopelitti** [comment]
I'm getting the same error on 0.26.1 / 0.25.x, RN 0.60.5 / 0.60.4.

Both physical device and emulator, Android 8.0.0.

Didn't try manual linking, though I'm assuming it would work.

It's working properly on iOS.

**Comment by rizwanatta** [comment]
ok! the comment is a bit vague!    can you tell me that are you trying an upgrade of the package in the project ! or is it a fresh project with packages libs and stuff! please! @eduardopelitti

**Comment by eduardopelitti** [comment]
Hi Rizwanatta, 

It's a fresh project with the versions I specified, I was originally on 0.60.4 and 0.26.1 and got this error. I tried downgrading to 0.25.0, that didn't work, then I tried Maps 0.26.1 and RN 0.60.5, didn't work either.

I followed the install steps for RN >0.60.

Let me know if you need more info. Thanks in advance.

**Comment by wmonecke** [comment]
Hi! I am having this issue on iOS on 0.26.1. Any ideas?

**Comment by RominHalltari** [comment]
I'm also having this issue on Android on 0.26.1 with react-native 0.60.5

**Comment by tinymobile** [comment]
Skippone config change fixed it for Android for me, but i'm still getting AIRMap not found" error on IOS.  Should that be changed for IOS as well?

**Comment by rizwanatta** [comment]
@tinymobile @wmonecke @RominHalltari @tinymobile @eduardopelitti   have you all tried for 

watch @skippone's config stuff in your config files and match them with yours from
YOUR_PROJECT/node_modules/react-native-maps/react-native-config.js

if they are same then do this, please!

1: `cd android && ./gradlew clean`, 
2:  uninstall the old installation of the application
3: 'react-native run-android'

on iOS devices

try building from XCODE after doing a Pod install

if you have already pods setup then
1:  `cd ios && pod deintegrate`
2:  `pod clean`
3:   `pod install`

in case you are using google map you have to add the import and your API KEY

feel free to reach out! 

FYI, :-) I just tested it on my new project of react-native 0.61 and 0.26.1 of react-native-maps! it works fine on both devices!

**Comment by ppv94** [comment]
I'm also having this issue on Android on 0.26.1 with react-native 0.60.5
Application crashes on open or android map are shown blank.
Which version of react-native-maps should I use

**Comment by JoeLeung32** [comment]
Still missing AIRMAP on my Android with RN@0.61.2, never give up, mate 👍

**Comment by javierfuentesm** [comment]
I still having the issue with the version v0.26.1

**Comment by anastely** [comment]
Hey @javierfuentesm , your issue in Android or iOS , if Android can you share the root build.gradle?

**Comment by rezasazesh** [comment]
Any update on this?

**Comment by phoenixxd** [comment]
This issue is still coming up.
    ```
"react": "^16.9.0",
    "react-native": "^0.61.5",
    "react-native-geolocation-service": "^4.0.2",
    "react-native-maps": "0.27.1",
```

**Comment by yuukiii** [comment]
> Any update on this?

https://github.com/react-native-community/react-native-maps/issues/2495#issuecomment-693178578

**Comment by eduardopelitti** [comment]
Perhaps this helps someone who's still struggling with a similar issue on Android.

In retrospect, my issue was due to an incorrect Android Setup, specifically with package auto linkage. 

I had a faulty setup in MainApplication.java, the result of adding packages that require modifying this file (React Native Navigation and CodePush), which was overriding auto linking and causing this issue (notice that in my case manual linking made the issue go away).

So specifically, in my project I had messed up with these lines when integrating RNN, resulting in broken autolinking:

```
@Override
  public List<ReactPackage> createAdditionalReactPackages() {
    return new PackageList(this).getPackages();
  }
```

Make sure `return new PackageList(this).getPackages();` is in place instead of > RN 0.60  `return Arrays.<ReactPackage>asList(`


Hope this points someone in the right direction.
